pages/learn/manage/organizations.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Organization management
+---
+
+# Organization management
+
+Organizations in balenaCloud are based around the idea of easier collaboration and robust management of members and applications. The feature blends in with current workflows all the while improving what is already there.
+
+## Sign up
```

Comment:
Maybe this feels a bit too obvious? I would probably just remove it.

Reply:
Roger that, added it because I was following the same template as account management document